addons/transform/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
Please don't commit eclipse settings

Reply:
I am used to more .gitirnore entries....